src/submission/forms.py

```diff
@@ -195,6 +196,12 @@ def __init__(self, *args, **kwargs):
                         except models.FieldAnswer.DoesNotExist:
                             pass
 
+                # if the editor is viewing the page, don't set additional
+                # fields to be required.
+                if editor_view:
+                    self.fields[element.name].required = False
+
+
```

Comment:
Does this belong on this branch? And does it need to be indented further so that it is part of the for loop?